pkg/apis/componentconfig/types.go

```diff
@@ -41,15 +41,6 @@ type DeschedulerConfiguration struct {
 	// Dry run
 	DryRun bool
 
-	// Node selectors
-	NodeSelector string
-
-	// MaxNoOfPodsToEvictPerNode restricts maximum of pods to be evicted per node.
-	MaxNoOfPodsToEvictPerNode int
-
-	// EvictLocalStoragePods allows pods using local storage to be evicted.
-	EvictLocalStoragePods bool
-
```

Comment:
I'd like to figure out our eventual plan for the componentconfig types. I started an issue for it a while back here https://github.com/kubernetes-sigs/descheduler/issues/431, but I don't know if it's just redundant with `Policy` or what

Reply:
Good point. I will put those back.